### PR TITLE
Jp/feature/create channel modal

### PR DESCRIPTION
**Pull request by adamkral12**

### CODE REVIEW

File: src/Channels/EditChannelModal.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            displayName: "",
+        };
+    }
+
+    onSubmit = () => {
+        this.props.onEdit(this.state);
+    };
+
+    handleNameChange = (event) => {
+        console.log(event);
```
Comment (kozaktomas) [inline src/Channels/EditChannelModal.jsx]: please remove console.log

File: src/Channels/InviteMemberModal.jsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                dataField: 'email',
+                formatter: this.renderButtons
+            },
+        ]
+
+        return (
+            <div>
+                <Modal show={this.props.show} onHide={this.props.onClose}>
+                    <Modal.Header closeButton>
+                        <Modal.Title>Edit channel name</Modal.Title>
```
Comment (adamkral12) [inline src/Channels/InviteMemberModal.jsx]: leftovers from copypasting

File: src/Channels/InviteMemberModal.jsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                    <Modal.Footer>
+                        <Button onClick={this.props.onClose}>Close</Button>
+                    </Modal.Footer>
+                </Modal>
+            </div>
+        );
+    }
+}
+/*
+{this.props.users.map((user) => {
```
Comment (adamkral12) [inline src/Channels/InviteMemberModal.jsx]: why so much commented code?

**Comment by adamkral12** [comment]
There are some errors about unused variables and required propTypes (you can view all of these in console) -> travis build is failing, check https://travis-ci.com/adamkral12/PV247

please watch out for these in the future

**Comment by adamkral12** [comment]
when I run npm install, there is diff in package-lock.json file, that means that you probably installed some modules localy, you can fix this by removing package-lock and runing install again

**Pull request merged**